cmd/devicesimulator/main.go

```diff
@@ -117,6 +136,28 @@ func main() {
 	log.Infoln("Simulator stopped.")
 }
 
+func reportVersion(versionFormat *string) {
+	cliVersion := version.Get()
+	switch *versionFormat {
+	case "":
+	case "yaml":
+		if marshalled, err := yaml.Marshal(&cliVersion); err == nil {
```

Comment:
do we care about this error? should we return if it exists?

Reply:
making exactly as in "version.go" then